examples/viewer.cpp

```diff
@@ -135,7 +150,7 @@ bool Viewer::render()
     // wipe the drawing surface clear
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-    GLint x = 0, y = 0, width = 600, height = 400;
```

Comment:
I'm no graphics expert but I don't see a logic connection here. Do you mean if a texture does not totally fit in the viewport, then the whole texture area is not updated?